.github/workflows/pull-request.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Count number of lines
         run: |
           chmod +x ./.github/workflows/countline.py
-          ./.github/workflows/countline.py --lines 600 --exclude_files src/screens/LoginPage/LoginPage.tsx src/GraphQl/Queries/Queries.ts src/screens/OrgList/OrgList.tsx src/GraphQl/Mutations/mutations.ts src/components/EventListCard/EventListCardModals.tsx
+          ./.github/workflows/countline.py --lines 600 --exclude_files src/screens/LoginPage/LoginPage.tsx src/GraphQl/Queries/Queries.ts src/screens/OrgList/OrgList.tsx src/GraphQl/Mutations/mutations.ts src/components/EventListCard/EventListCardModals.tsx src/screens/ManageTag/ManageTag.tsx
```

Comment:
Doesn't seem like we should ignore these files? Is this future plan to restrict it to 600 lines?

Reply:
I think this check was added in the months before this gsoc... I'm not sure about how 600 lines was decided upon. This could be discussed and changed as the app grows and includes more features that would cause files to be larger than 600 lines. Still I'll try to reduce this file's size in the upcoming PRs for refactoring...